src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -1189,13 +1186,10 @@ static void Main()
                 Assert.Equal(0, parameters[3].GetAttributes(paramAttrType).Count());
 
                 // verify no ParamArrayAttribute on p2
-                if (isFromSource)
-                {
-                    WellKnownAttributesTestBase.VerifyParamArrayAttribute(parameters[1], (SourceModuleSymbol)moduleSymbol, expected: false);
-                }
+                VerifyParamArrayAttribute(parameters[1], expected: false);
             };
 
-            CompileAndVerify(source, sourceSymbolValidator: symbolValidator(true), symbolValidator: symbolValidator(false));
+            CompileAndVerify(source, symbolValidator: symbolValidator);
```

Comment:
>CompileAndVerify(source, symbolValidator: symbolValidator); [](start = 12, length = 59)

This change removes testing for source, I think we should put it back.

Reply:
I mean we should still pass sourceSymbolValidator.

---
In reply to: [150616316](https://github.com/dotnet/roslyn/pull/22896#discussion_r150616316) [](ancestors = 150616316)